src/store/state/state-model.ts

```diff
@@ -57,6 +56,7 @@ export const stateModel: StateModelIF = {
       isValidSpecialResolution: true,
       isValidSpecialResolutionSignature: true,
       isValidApprovalType: true,
+      isValidRelationship: true,
```

Comment:
I found this bug while testing my other changes: console errors due to an element id not found (null de-reference).

The order here is now correct. I also updated some comments above.

This is also how I found out that `Element.scrollIntoView()` is not async and so I cleaned that up, too.